django/core/mail/message.py

```diff
@@ -263,9 +268,6 @@ def message(self):
         msg = self._create_message(msg)
         msg["Subject"] = self.subject
         msg["From"] = self.extra_headers.get("From", self.from_email)
-        self._set_list_header_if_not_empty(msg, "To", self.to)
-        self._set_list_header_if_not_empty(msg, "Cc", self.cc)
-        self._set_list_header_if_not_empty(msg, "Reply-To", self.reply_to)
```

Comment:
`_set_list_header_if_not_empty` is now unused in the class. Could we still use it? Or should it be deleted? (but that should maybe be documented then)

Reply:
> `_set_list_header_if_not_empty` is now unused in the class. Could we still use it? Or should it be deleted? (but that should maybe be documented then)

I don't know if it can still be useful, but I could modify that function for doing the same thing instead of delete it. For example I can modify `_set_list_header_if_not_empty` in this way

```python
def _set_list_header_if_not_empty(self, header, values, msg):
  if self.extra_headers.get(header) is None:
     if values:
         msg[header] = ", ".join(str(v) for v in values)
```

and then use it 

```python
for header, values_header in self.headers_with_one_occurrence.items():
   _set_list_header_if_not_empty(self, header, values_header, msg)
```

What do you think?

Reply:
Seems unnecessary indeed.

I'd suggest deleting it :) Since it's an [internal API](https://docs.djangoproject.com/en/5.0/misc/api-stability/#apis-marked-as-internal), I think there is no need to document the change.